web/libs/editor/src/tags/object/RichText/domManager.ts

```diff
@@ -366,7 +368,8 @@ class DomData {
     let fromIdx = this.displayedTextPos;
     const contentParts = [];
 
-    while (displayedText[fromIdx] === LF) {
+    // it should be just LF but in some OS / browsers it's LF and CR
```

Comment:
From this comment it looks like different OS / browsers have different inner content for the same text, right? And in this case browsers with CRLF will have +1 extra symbol for every new line, producing shifted indices, which will be inconsistent in other OS / browsers and that's a huge risk. maybe we have to check for `CRLF|LF|CR` generating one symbol for CRLF(s)

Reply:
Thank you. I'll fix related comments. But for the context here: it's inner mostly counter that do not affect positions in result, but it can provide visual isues with displayed text (in region details) as it will lead to converting /r/n/r/n into /n/n/n/n. But it's historically done thing.